src/components/Contacts/ContactFilters/ContactFilters.stories.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { ReactElement } from 'react';
+import { ContactFilters } from './ContactFilters';
+
+export default {
+  title: 'ContactFilters',
+};
+
+const accountListId = '111';
+
+export const Default = (): ReactElement => {
+  return <ContactFilters accountListId={accountListId} />;
+};
+
+export const Loading = (): ReactElement => {
+  return <ContactFilters accountListId={accountListId} />;
```

Comment:
@OzzieOrca Is there a way I can mock the GraphQL results for Storybook to display this component in different states?  Same for the ContactsTable.

Reply:
@reldredge71  I believe you can wrap it in a ```<MockedProvider>``` and then pass your mocks you want to it, Here's an example: https://github.com/CruGlobal/mpdx-react/blob/811726cf039830f5449da17790f2ec28f6c6aa7b/src/components/Task/Drawer/ContactList/ContactList.stories.tsx#L15-L18

And if you wanted to do a loading state you would do something like this:
https://github.com/CruGlobal/mpdx-react/blob/811726cf039830f5449da17790f2ec28f6c6aa7b/src/components/Task/Drawer/ContactList/ContactList.stories.tsx#L29-L34

Reply:
@TheNoodleMoose thanks!  Another question: I want to display the filters panel before and after the "load filters" button is pushed, because the loading state doesn't appear until after the button triggers the lazy query.  Is there a way to simulate a button push in order to display the loading state in Storybook?  (Maybe I can leave that alone for now and let @frett do it when he work on the filters)

Reply:
I'm fine with you leaving that for me if you want

Reply:
Hmm, I'm not sure as far as storybook how to do that or if it's possible(yet). Maybe @OzzieOrca would know. But maybe a unit test would be better for that scenario? You have [this test](https://github.com/CruGlobal/mpdx-react/pull/34/files/e9416cccab9c79483b5dcdeea01d100c860f6386#diff-2f3a5e662d909bbf6e4ed9179d38312e1b7e8f288c861584499ca26a61d1fec4R26) which I believe covers the test case. But at the same time, it would be nice to visually see it happen.

Reply:
Yeah I would like to visually show the before, during, and after loading states for the design team's approval, hopefully we find a way to do that.  So far it looks like Storybook will give us outputs for the provided inputs, but triggering things inside the component is different.

Reply:
Yeah, I tried looking at the docs but didn't see anything about mocking data and waiting for that data to be provided to the component until something is interacted with.